debian/changelog

```diff
@@ -1,3 +1,9 @@
+system76-driver (20.04.53~~alpha) focal; urgency=low
+
+  * Daily WIP for 20.04.53
```

Comment:
Don't forget the changelog entry under the Daily WIP line

Reply:
Done.